src/components/chart-wrapper/index.js

```diff
@@ -18,23 +18,32 @@ const Title = styled('div', forwardRef)(
   (props) => `
     margin: 0px 16px 8px 16px;
     overflow-wrap: anywhere;
-    color: ${ props['title-props'].color };
-    font-size: ${ props['title-props'].fontSize };
-    font-weight: ${ props['title-props'].fontWeight };
-    text-align: ${ props['title-props'].textAlign };
+    color: ${ props.color };
+    font-size: ${ props['font-size'] };
+    font-weight: ${ props['font-weight'] };
+    text-align: ${ props['text-align'] };
```

Comment:
how are undefined cases handled when one doesn't want to specify certain props? should we have safeguard values to fall back to?

Reply:
_preference:_ for such a straightforward relay to the style, unless these props are particularly important, I like `{ ...titleStyle }`. It's then just a catch-all. These are also less likely to change, so maybe even `style={titleStyle}`?

Reply:
> how are undefined cases handled when one doesn't want to specify certain props? should we have safeguard values to fall back to?

Yes, I had default props defined. But, in light of what Shane proposed, which was more in line with what I wanted initially to do as well, I separated styling from title and, if the user wants to change the styling, we require that he specifies all styling.